documentation/src/main/asciidoc/query.asciidoc

```diff
@@ -800,6 +824,104 @@ Be aware that fields used for sorting must not be tokenized (see <<field-annotat
 be marked as sortable field using the `@SortableField` annotation (see <<sortablefield-annotation>>).
 ====
 
+
+[WARNING]
+====
+In the special case of numeric fields indexed through *custom* ``FieldBridge``s, you must provide the SortField type along with the field name.
+This does not apply to numeric fields using default field bridges.
+=====
+[source, JAVA]
+-----
+QueryBuilder builder = fullTextSession.getSearchFactory()
+    .buildQueryBuilder().forEntity(Book.class).get();
+Query luceneQuery = /* ... */;
+FullTextQuery query = s.createFullTextQuery( luceneQuery, Book.class );
+Sort sort = builder
+  .sort()
+    .byField("authorId", SortField.Type.LONG) // Sort type provided HERE
+    .andByField("title")
+  .createSort();
+query.setSort(sort);
+List results = query.list();
+-----
+=====
+====
+
+====== Handling missing values
+
+By default, Hibernate Search will expect every document to contain exactly one value for the given sort fields. If it's not the case, you may decide whether documents with missing values will end up in first or last position in the result list:
+
+.Returning documents with missing values in first position when sorting
+====
+[source, JAVA]
+----
+QueryBuilder builder = fullTextSession.getSearchFactory()
+    .buildQueryBuilder().forEntity(Book.class).get();
+Query luceneQuery = /* ... */;
+FullTextQuery query = s.createFullTextQuery( luceneQuery, Book.class );
+Sort sort = builder
+  .sort()
+    .byField("author")
+      .onMissingValue().sortFirst()
+    .andByField("title")
+  .createSort();
+query.setSort(sort);
+List results = query.list();
+----
+====
+
+For numeric fields (and only numeric fields), you may even provide a default value to be used in case it's missing:
```

Comment:
Both, actually. It's a limitation of `org.apache.lucene.search.SortField.setMissingValue(Object)`, which prevents us from setting the missing value for `String` fields. Since we're using `Sort`/`SortField` for carrying information, including with the Elasticsearch backend, the Lucene limitation becomes ours.